WcaOnRails/app/views/registrations/_register_form.html.erb

```diff
@@ -12,7 +12,8 @@
   <%= f.input :comments, disabled: disabled %>
 
   <%# If the user has to pay fee here, this information will already be in the payment form %>
-  <% unless @registration.has_to_pay_fee_here? %>
+  <%# If there are no entry fees, no need to diplay this information %>
+  <% unless @registration.has_to_pay_fee_here? || @registration.entry_fee == 0 %>
```

Comment:
1. `has_to_pay_fee_here?` needs a name that makes clear what/when/where "here" means.

2. Should maybe the ` || @registration.entry_fee == 0` part be _in_ that function?